lib/FrameManager.js

```diff
@@ -777,6 +778,7 @@ class WaitTask {
     this._polling = polling;
     this._timeout = timeout;
     this._predicateBody = helper.isString(predicateBody) ? 'return ' + predicateBody : 'return (' + predicateBody + ')(...args)';
+    this._title = title;
```

Comment:
there's no need to save the `title`

Reply:
There is also no need to save `this._timeout`, but it's helpful when using a debugger to see what the WaitTask does

Reply:
`this._timeout` is used in `rerun()` method.

> but it's helpful when using a debugger to see what the WaitTask does

I don't think it's a big of a deal: it's easy to save title once the debugging need occurs.